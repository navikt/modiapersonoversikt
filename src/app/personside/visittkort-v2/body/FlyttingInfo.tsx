import { GlobeFillIcon } from '@navikt/aksel-icons';
import { Box } from '@navikt/ds-react';
import { Normaltekst } from 'nav-frontend-typografi';
import type { Person } from 'src/app/personside/visittkort-v2/PersondataDomain';
import Endringstekst from 'src/app/personside/visittkort-v2/body/Endringstekst';
import GyldighetsPeriode from 'src/app/personside/visittkort-v2/body/GyldighetsPeriode';
import VisittkortElement from 'src/app/personside/visittkort-v2/body/VisittkortElement';
import { VisittkortGruppe } from 'src/app/personside/visittkort-v2/body/VisittkortStyles';
import { formaterDato } from 'src/utils/string-utils';

interface Props {
    person: Person;
}

export default function FlyttingInfo({ person }: Props) {
    if (person.innflyttingTilNorge.isEmpty() && person.utflyttingFraNorge.isEmpty()) {
        return null;
    }
    return (
        <VisittkortGruppe tittel="Flytting" ikon={<GlobeFillIcon color="#C6C2BF" />}>
            {person.innflyttingTilNorge.isNotEmpty() && (
                <VisittkortElement beskrivelse="Flyttet fra">
                    {person.innflyttingTilNorge.map((innFlytting, index) => {
                        return (
                            <Box key={`${innFlytting.gyldighetsPeriode}-${index}`} className="mb-[0.5rem]">
                                <Normaltekst>{innFlytting.fraflyttingsland} </Normaltekst>
                                {innFlytting.sistEndret && <Endringstekst sistEndret={innFlytting.sistEndret} />}
                                {innFlytting.gyldighetsPeriode && (
                                    <GyldighetsPeriode gyldighetsPeriode={innFlytting.gyldighetsPeriode} />
                                )}
                            </Box>
                        );
                    })}
                </VisittkortElement>
            )}
            {person.utflyttingFraNorge.isNotEmpty() && (
                <VisittkortElement beskrivelse="Flyttet til">
                    {person.utflyttingFraNorge.map((utflyttingFraNorge, index) => {
                        return (
                            <Box key={`${utflyttingFraNorge.gyldighetsPeriode}-${index}`} className="mb-[0.5rem]">
                                <Normaltekst>{utflyttingFraNorge.tilflyttingsland} </Normaltekst>
                                {utflyttingFraNorge.utflyttingsdato && (
                                    <Normaltekst>
                                        Utflyttingsdato: {formaterDato(utflyttingFraNorge.utflyttingsdato)}
                                    </Normaltekst>
                                )}
                                {utflyttingFraNorge.sistEndret && (
                                    <Endringstekst sistEndret={utflyttingFraNorge.sistEndret} />
                                )}
                                {utflyttingFraNorge.gyldighetsPeriode && (
                                    <GyldighetsPeriode gyldighetsPeriode={utflyttingFraNorge.gyldighetsPeriode} />
                                )}
                            </Box>
                        );
                    })}
                </VisittkortElement>
            )}
        </VisittkortGruppe>
    );
}
