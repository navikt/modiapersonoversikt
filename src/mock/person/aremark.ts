import { Diskresjonskoder, TilrettelagtKommunikasjonsTyper } from '../../konstanter';
import { Kjønn, Person, Relasjonstype, SivilstandTyper } from '../../models/person/person';

export const aremark: Person = {
    fødselsnummer: '10108000398',
    kjønn: Kjønn.Mann,
    geografiskTilknytning: '0118',
    alder: 42,
    navn: {
        sammensatt: 'AREMARK TESTFAMILIEN',
        fornavn: 'AREMARK',
        mellomnavn: '',
        etternavn: 'TESTFAMILIEN',
    },
    tilrettelagtKomunikasjonsListe: [
        {
            behovKode: 'TOHJ',
            beskrivelse: TilrettelagtKommunikasjonsTyper.TOHJ
        }
    ],
    diskresjonskode: Diskresjonskoder.FORTROLIG_ADRESSE,
    statsborgerskap: 'NORSK',
    personstatus: {
        dødsdato: undefined,
        bostatus: undefined
    },
    sivilstand: {
        value: SivilstandTyper.Gift,
        beskrivelse: 'Gift',
        fraOgMed: '2005-12-12'
    },
    familierelasjoner: [{
        harSammeBosted: false,
        rolle: Relasjonstype.Barn,
        tilPerson: {
            navn: {
                sammensatt: '',
                fornavn: 'Siri',
                mellomnavn: null,
                etternavn: 'Aremark'
            },
            alder: 17,
            fødselsnummer: '10117100000',
            personstatus: {}
        }

    }],
    kontaktinformasjon: {
        mobil: {
            sistEndret: '2014-06-21T18:44:39+02:00',
            telefonnummer: '99887766',
            retningsnummer: '47',
            sistEndretAv: 'BRUKER'
        }
    }
};