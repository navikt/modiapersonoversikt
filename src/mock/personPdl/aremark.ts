import {
    Adresse,
    AdresseBeskyttelse,
    EgenAnsatt,
    Enhet,
    Kjonn,
    KodeBeskrivelse,
    Navn,
    Person,
    PersonStatus,
    Sikkerhetstiltak,
    SikkerhetstiltakType,
    Sivilstand,
    SivilstandType,
    Statsborgerskap,
    Telefon,
    TilrettelagtKommunikasjon
} from '../../models/personPdl/person';
import { getAremarkForeldreansvar } from './foreldreansvarMock';
import { getDeltBostedMock } from './deltBostedMock';
import * as faker from 'faker';
import { getMockDodsbo } from './dodsbo';
import navfaker from 'nav-faker/dist/index';
import { mockVergemal } from './vergemalMock';
import { DigitalKontaktinformasjon } from '../../models/personPdl/digitalKontaktinformasjon';
import { getBankKonto } from './bankkontoMock';
import { Fullmakt, FullmaktsRolle } from '../../models/personPdl/fullmakt';

export const navnAremark: Navn = {
    fornavn: 'TESTFAMILIEN',
    mellomnavn: null,
    etternavn: 'AREMARK'
};

export const kodeBeskrivelseKjonn: KodeBeskrivelse<Kjonn> = {
    kode: Kjonn.Mann,
    beskrivelse: 'Mann'
};

export const adresseAremark: Adresse = {
    linje1: 'Adresseveien 1, 0661 AREMARKSTAD',
    linje2: null,
    linje3: null
};

export const navEnhetAremark: Enhet = {
    id: '0000',
    navn: 'Nav-kontor'
};

export const statsborgerskapAremark: Statsborgerskap = {
    land: {
        kode: 'NOR',
        beskrivelse: 'Norsk statsborger'
    },
    gyldigFraOgMed: new Date('1980-10-10'),
    gyldigTilOgMed: null
};

export const adressebeskyttelseAremark: KodeBeskrivelse<AdresseBeskyttelse> = {
    kode: AdresseBeskyttelse.Ugradert,
    beskrivelse: 'Ingen'
};

export const sikkerhetstiltakAremark: Sikkerhetstiltak = {
    type: SikkerhetstiltakType.TelefoniskUtestengelse,
    gyldigFraOgMed: new Date('2005-02-13'),
    gyldigTilOgMed: new Date('2020-02-02')
};

export const personstatusAremark: KodeBeskrivelse<PersonStatus> = {
    kode: PersonStatus.Bosatt,
    beskrivelse: 'Bosatt i Norge'
};

export const sivilstandAremark: Sivilstand = {
    type: {
        kode: SivilstandType.Skilt,
        beskrivelse: 'Skilt'
    },
    gyldigFraOgMed: new Date('2010-03-06')
};

export const tilrettelagtKommunikasjonAremark: TilrettelagtKommunikasjon = {
    talesprak: [
        {
            kode: 'NO',
            beskrivelse: 'Norsk'
        }
    ],
    tegnsprak: [
        {
            kode: 'NO',
            beskrivelse: 'Norsk'
        }
    ]
};

export const telefonAremark: Telefon = {
    retningsnummer: {
        kode: '+47',
        beskrivelse: 'retningsnummer er norsk'
    },
    identifikator: '99009900',
    sistEndret: new Date('2017-10-10'),
    sistEndretAv: 'TESTFAMILIEN AREMARK',
    prioritet: 1
};

export const digitalKontaktInformasjonAremark: DigitalKontaktinformasjon = {
    personident: '10108000398',
    reservasjon: 'reservert',
    epostadresse: {
        value: null,
        sistOppdatert: null,
        sisVerifisert: new Date('2013-01-01')
    },
    mobiltelefonnummer: {
        value: null,
        sistOppdatert: null,
        sisVerifisert: null
    }
};

export const fullmaktAremark: Fullmakt = {
    motpartsPersonident: '123456789',
    motpartsPersonNavn: {
        fornavn: 'Navn',
        mellomnavn: null,
        etternavn: 'Navnesen'
    },
    motpartsRolle: FullmaktsRolle.Fullmektig,
    omraade: ['*'],
    gyldigFraOgMed: new Date('2019-01-01'),
    gyldigTilOgMed: new Date('2022-12-12')
};

export const aremark: Person = {
    fnr: '10108000398',
    navn: [navnAremark],
    kjonn: [kodeBeskrivelseKjonn],
    fodselsdato: [new Date('1980-10-10')],
    dodsdato: [],
    bostedAdresse: [adresseAremark],
    kontaktAdresse: [adresseAremark],
    navEnhet: navEnhetAremark,
    statsborgerskap: [statsborgerskapAremark],
    adressebeskyttelse: [adressebeskyttelseAremark],
    sikkerhetstiltak: [sikkerhetstiltakAremark],
    erEgenAnsatt: EgenAnsatt.Nei,
    personstatus: [personstatusAremark],
    sivilstand: [sivilstandAremark],
    foreldreansvar: [getAremarkForeldreansvar()],
    deltBosted: getDeltBostedMock(faker),
    dodsbo: getMockDodsbo(faker),
    fullmakt: [fullmaktAremark],
    vergemal: mockVergemal(navfaker.personIdentifikator.f√∏dselsnummer()),
    tilrettelagtKommunikasjon: tilrettelagtKommunikasjonAremark,
    telefonnummer: [telefonAremark],
    kontaktOgReservasjon: digitalKontaktInformasjonAremark,
    bankkonto: getBankKonto()
};
