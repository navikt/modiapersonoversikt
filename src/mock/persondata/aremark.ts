import {
    AdresseBeskyttelse,
    EgenAnsatt,
    ForelderBarnRelasjon,
    ForelderBarnRelasjonRolle,
    FullmaktsRolle,
    Kjonn,
    LocalDate,
    LocalDateTime,
    Person,
    PersonStatus,
    SivilstandType,
    Skifteform
} from '../../app/personside/visittkort-v2/PersondataDomain';
import { harDiskresjonskode } from '../../app/personside/visittkort-v2/visittkort-utils';

export function hentBarnAremark(): ForelderBarnRelasjon[] {
    return [
        {
            ident: '12345678910',
            rolle: ForelderBarnRelasjonRolle.BARN,
            navn: [
                {
                    fornavn: 'BARN1',
                    mellomnavn: null,
                    etternavn: 'BARNESEN'
                }
            ],
            fodselsdato: ['2008-03-15' as LocalDate],
            alder: 13,
            kjonn: [
                {
                    kode: Kjonn.K,
                    beskrivelse: 'Kvinne'
                }
            ],
            adressebeskyttelse: [
                {
                    kode: AdresseBeskyttelse.UGRADERT,
                    beskrivelse: 'UGRADERT'
                }
            ],
            harSammeAdresse: false,
            dodsdato: []
        },
        {
            ident: '12345678911',
            rolle: ForelderBarnRelasjonRolle.BARN,
            navn: [
                {
                    fornavn: 'BARN1',
                    mellomnavn: null,
                    etternavn: 'BARNESEN'
                }
            ],
            fodselsdato: ['2008-03-15' as LocalDate],
            alder: null,
            kjonn: [
                {
                    kode: Kjonn.K,
                    beskrivelse: 'Kvinne'
                }
            ],
            adressebeskyttelse: [],
            harSammeAdresse: false,
            dodsdato: []
        }
    ];
}

export const aremark: Person = {
    fnr: '10108000398',
    navn: [
        {
            fornavn: 'AREMARK',
            mellomnavn: null,
            etternavn: 'TESTFAMILIEN'
        }
    ],
    kjonn: [
        {
            kode: Kjonn.M,
            beskrivelse: 'Mann'
        }
    ],
    fodselsdato: ['1980-10-10' as LocalDate],
    geografiskTilknytning: '0118',
    alder: 41,
    dodsdato: [],
    bostedAdresse: [
        {
            linje1: 'Islandsgate 49',
            linje2: '7693 Hovedøya',
            linje3: 'Norge',
            sistEndret: {
                ident: 'Folkeregisteret',
                tidspunkt: '2010-02-20T10:15:30' as LocalDateTime,
                system: 'FREG'
            },
            gyldighetsPeriode: null
        }
    ],
    kontaktAdresse: [
        {
            linje1: 'Aremarkveien 1',
            linje2: '0320 Aremark',
            linje3: null,
            sistEndret: {
                ident: 'Folkeregisteret',
                tidspunkt: '2020-01-01T10:15:30' as LocalDateTime,
                system: 'FREG'
            },
            gyldighetsPeriode: null
        }
    ],
    oppholdsAdresse: [
        {
            linje1: 'Oppholdsadresse 1',
            linje2: '0000 AREMARK',
            linje3: null,
            sistEndret: null,
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-02-01' as LocalDate,
                gyldigTilOgMed: '2022-05-01' as LocalDate
            }
        }
    ],
    navEnhet: {
        id: '0219',
        navn: 'NAV Bærum',
        publikumsmottak: [
            {
                besoksadresse: {
                    linje1: 'Adressevei 1',
                    linje2: '0000 AREMARK',
                    linje3: null,
                    sistEndret: null,
                    gyldighetsPeriode: null
                },
                apningstider: [
                    {
                        ukedag: 'Mandag',
                        apningstid: '09.00 - 15.00'
                    },
                    {
                        ukedag: 'Tirsdag',
                        apningstid: '09.00 - 15.00'
                    },
                    {
                        ukedag: 'Onsdag',
                        apningstid: '10.00 - 16.00'
                    }
                ]
            }
        ]
    },
    statsborgerskap: [
        {
            land: {
                kode: 'NOR',
                beskrivelse: 'NORGE'
            },
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-02-01' as LocalDate,
                gyldigTilOgMed: '2022-05-01' as LocalDate
            }
        }
    ],
    adressebeskyttelse: [
        {
            kode: AdresseBeskyttelse.KODE6,
            beskrivelse: 'Kode 6'
        }
    ],
    sikkerhetstiltak: [
        {
            type: 'TFUS',
            beskrivelse: 'Telefonisk utestengelse',
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-02-01' as LocalDate,
                gyldigTilOgMed: '2022-05-01' as LocalDate
            }
        }
    ],
    erEgenAnsatt: EgenAnsatt.JA,
    personstatus: [
        {
            kode: PersonStatus.BOSATT,
            beskrivelse: 'Bosatt i Norge'
        }
    ],
    sivilstand: [
        {
            type: {
                kode: SivilstandType.GIFT,
                beskrivelse: 'Gift'
            },
            gyldigFraOgMed: '2005-12-12' as LocalDate,
            sivilstandRelasjon: {
                fnr: '12345555555',
                navn: [
                    {
                        fornavn: 'BAREMARK',
                        mellomnavn: null,
                        etternavn: 'TESTFAMILIEN'
                    }
                ],
                alder: 40,
                adressebeskyttelse: [
                    {
                        kode: AdresseBeskyttelse.UGRADERT,
                        beskrivelse: 'UGRADERT'
                    }
                ],
                harSammeAdresse: true
            }
        }
    ],
    foreldreansvar: hentBarnAremark().map((barn) => ({
        ansvar: 'felles',
        ansvarlig: null,
        ansvarsubject: {
            navn: harDiskresjonskode(barn.adressebeskyttelse) ? null : barn.navn.firstOrNull(),
            ident: barn.ident
        }
    })),
    deltBosted: [
        {
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-02-01' as LocalDate,
                gyldigTilOgMed: '2022-05-01' as LocalDate
            },
            adresse: {
                linje1: `Adresseveien 1`,
                linje2: '0000 Aremark',
                linje3: null,
                sistEndret: null,
                gyldighetsPeriode: null
            }
        }
    ],
    dodsbo: [
        {
            adressat: {
                advokatSomAdressat: {
                    kontaktperson: {
                        fornavn: 'Advokat',
                        mellomnavn: null,
                        etternavn: 'Navnesen'
                    },
                    organisasjonsnavn: 'Advokatkontoret Aremark',
                    organisasjonsnummer: '01234567891011'
                },
                organisasjonSomAdressat: null,
                personSomAdressat: null
            },
            adresse: {
                linje1: 'Elgelia 20, 0000 Aremark',
                linje2: null,
                linje3: null,
                sistEndret: null,
                gyldighetsPeriode: null
            },
            registrert: '2010-02-02' as LocalDate,
            skifteform: Skifteform.OFFENTLIG,
            sistEndret: {
                ident: 'Folkeregisteret',
                tidspunkt: '2020-01-01T10:15:30' as LocalDateTime,
                system: 'Folkeregisteret'
            }
        }
    ],
    fullmakt: [
        {
            motpartsPersonident: '123456789',
            motpartsPersonNavn: {
                fornavn: 'Navn',
                mellomnavn: null,
                etternavn: 'Navnesen'
            },
            motpartsRolle: FullmaktsRolle.FULLMEKTIG,
            omrade: [
                {
                    kode: '*',
                    beskrivelse: ''
                }
            ],
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-06-01' as LocalDate,
                gyldigTilOgMed: '2022-05-01' as LocalDate
            }
        }
    ],
    vergemal: [
        {
            ident: '21042900076',
            navn: {
                fornavn: 'Simen',
                mellomnavn: null,
                etternavn: 'Solli'
            },
            vergesakstype: 'Voksen',
            omfang: 'Ivareta personens interesser innenfor det personlige og økonomiske området',
            embete: 'Fylkesmannen i Troms og Finnmark',
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-06-01' as LocalDate,
                gyldigTilOgMed: '2022-05-10' as LocalDate
            }
        },
        {
            ident: '123456799',
            navn: {
                fornavn: 'Truls',
                mellomnavn: null,
                etternavn: 'Tøffel'
            },
            vergesakstype: 'Fremtidsfullmakt',
            omfang: 'Ivareta personens interesser innenfor det økonomiske området',
            embete: 'Fylkesmannen i Troms og Finnmark',
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-02-10' as LocalDate,
                gyldigTilOgMed: '2022-05-01' as LocalDate
            }
        }
    ],
    tilrettelagtKommunikasjon: {
        talesprak: [
            {
                kode: 'NO',
                beskrivelse: 'Norsk'
            }
        ],
        tegnsprak: [
            {
                kode: 'NO',
                beskrivelse: 'Norsk'
            }
        ]
    },
    telefonnummer: [
        {
            retningsnummer: {
                kode: '+47',
                beskrivelse: 'Norge'
            },
            identifikator: '99009900',
            sistEndret: {
                ident: '11223344',
                tidspunkt: '2018-06-14T00:00:00' as LocalDateTime,
                system: 'BD06'
            },
            prioritet: -1
        }
    ],
    kontaktOgReservasjon: {
        personident: '10108000398',
        reservasjon: 'true',
        epostadresse: {
            value: 'epost@nav.no',
            sistOppdatert: null,
            sistVerifisert: '2013-01-01' as LocalDate
        },
        mobiltelefonnummer: {
            value: '12345678',
            sistOppdatert: '2021-11-02' as LocalDate,
            sistVerifisert: '2021-11-02' as LocalDate
        }
    },
    bankkonto: {
        kontonummer: '12345678900',
        banknavn: 'DNB ASA',
        sistEndret: {
            ident: '1010800 BD03',
            tidspunkt: '2006-03-15T00:00:00' as LocalDateTime,
            system: ''
        },
        bankkode: null,
        swift: 'DNBANOKKXXX',
        landkode: null,
        adresse: {
            linje1: 'Bankveien 1,',
            linje2: '0357 Bankestad',
            linje3: null,
            sistEndret: null,
            gyldighetsPeriode: null
        },
        valuta: {
            kode: 'NOK',
            beskrivelse: 'Norske kroner'
        }
    },
    forelderBarnRelasjon: hentBarnAremark()
};
