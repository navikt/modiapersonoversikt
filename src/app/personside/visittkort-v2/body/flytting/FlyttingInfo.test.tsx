import { act } from '@testing-library/react';
import FlyttingInfo from 'src/app/personside/visittkort-v2/body/flytting/FlyttingInfo';
import { aremark } from 'src/mock/persondata/aremark';
import { renderWithProviders } from 'src/test/Testprovider';

test('viser info om flytting', async () => {
    const visittkortbody = await act(() => renderWithProviders(<FlyttingInfo person={aremark} />));
    expect(visittkortbody.asFragment()).toMatchSnapshot();
});
