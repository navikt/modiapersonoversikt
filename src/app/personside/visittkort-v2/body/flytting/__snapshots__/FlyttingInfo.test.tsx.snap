// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`viser info om flytting 1`] = `
<DocumentFragment>
  .c4 {
  color: #645f5a;
  margin-top: 3px;
}

.c4 .typo-etikett-liten {
  line-height: 1rem;
}

.c0 {
  position: relative;
  display: flex;
  align-items: center;
}

.c1 {
  position: absolute;
  left: -3.125rem;
  width: 3.125rem;
  display: flex;
  justify-content: center;
}

.c1 >svg {
  height: 1.5rem;
  width: auto;
}

.c3 {
  margin: 10px 0 20px 0;
}

.c3:last-child {
  margin: 10px 0 0 0;
}

.c2 {
  margin-bottom: 1rem;
}

<section>
    <div
      class="c0"
    >
      <div
        class="c1"
      >
        <svg
          color="#C6C2BF"
          fill="none"
          focusable="false"
          height="1em"
          role="img"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M15.033 2.731a.242.242 0 0 0-.308.29l.008.033c.401 1.723.691 3.367.873 4.973a.25.25 0 0 0 .25.223h4.772c.178 0 .299-.182.224-.343a9.78 9.78 0 0 0-5.819-5.176m-2.197-.446a.25.25 0 0 1 .218.19l.219.92c.369 1.587.64 3.1.817 4.577a.25.25 0 0 1-.248.278h-3.685a.25.25 0 0 1-.247-.278c.176-1.477.448-2.99.817-4.577l.219-.92a.25.25 0 0 1 .217-.19 10 10 0 0 1 1.673 0m-3.87.446a.242.242 0 0 1 .308.29l-.007.03v.003a43 43 0 0 0-.874 4.973.25.25 0 0 1-.249.223H3.372a.243.243 0 0 1-.224-.343A9.78 9.78 0 0 1 8.966 2.73M2.71 9.75a.246.246 0 0 0-.242.193 9.8 9.8 0 0 0 0 4.114.246.246 0 0 0 .242.193h5.264a.25.25 0 0 0 .25-.266 35 35 0 0 1 0-3.968.25.25 0 0 0-.25-.266zm.663 6a.243.243 0 0 0-.224.343 9.78 9.78 0 0 0 5.82 5.176c.184.06.355-.11.305-.296l-.004-.014-.003-.013c-.4-1.716-.69-3.36-.873-4.973a.25.25 0 0 0-.25-.223zm7.606 5.785c.027.1.113.174.217.182a10 10 0 0 0 1.61 0 .25.25 0 0 0 .216-.182l.254-.941c.367-1.577.639-3.087.815-4.566a.25.25 0 0 0-.247-.278h-3.686a.25.25 0 0 0-.248.278c.177 1.48.449 2.989.815 4.566zm3.748-.562a.241.241 0 0 0 .306.296 9.78 9.78 0 0 0 5.82-5.176.243.243 0 0 0-.224-.343h-4.772a.25.25 0 0 0-.25.223 43 43 0 0 1-.873 4.973l-.003.013zm6.565-6.723c.116 0 .217-.08.242-.193a9.8 9.8 0 0 0 0-4.114.246.246 0 0 0-.242-.193h-5.264a.25.25 0 0 0-.25.266c.076 1.33.075 2.644 0 3.968a.25.25 0 0 0 .25.266zm-7.02-.233a.25.25 0 0 1-.248.233H9.976a.25.25 0 0 1-.248-.233 33 33 0 0 1 0-4.034.25.25 0 0 1 .249-.233h4.046c.132 0 .24.102.249.233a33 33 0 0 1 0 4.034"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </div>
      <h3
        class="typo-undertittel"
      >
        Flytting
      </h3>
    </div>
    <div
      class="c2"
    />
    <div
      class="c3"
    >
      <div
        class="c0"
      >
        <h4
          class="typo-element"
        >
          Flyttet fra
        </h4>
      </div>
      <div
        class="navds-box mb-[0.5rem]"
      >
        <p
          class="typo-normal"
        >
          Sverige 
        </p>
        <div
          class="c4"
        >
          <p
            class="typo-undertekst"
          >
            Endret 20.02.2010 av folkeregisteret (kilde: bruker)
          </p>
        </div>
        <div
          class="c4"
        >
          <p
            class="typo-undertekst"
          >
            Gyldig: 01.01.2020 – 01.01.2030
          </p>
        </div>
      </div>
      <div
        class="navds-box mb-[0.5rem]"
      >
        <p
          class="typo-normal"
        >
          Hellas 
        </p>
        <div
          class="c4"
        >
          <p
            class="typo-undertekst"
          >
            Endret 20.02.2010 av folkeregisteret (kilde: bruker)
          </p>
        </div>
        <div
          class="c4"
        >
          <p
            class="typo-undertekst"
          >
            Gyldig: 01.01.2020 – 01.01.2030
          </p>
        </div>
      </div>
    </div>
    <div
      class="c3"
    >
      <div
        class="c0"
      >
        <h4
          class="typo-element"
        >
          Flyttet til
        </h4>
      </div>
      <div
        class="navds-box mb-[0.5rem]"
      >
        <p
          class="typo-normal"
        >
          Danmark 
        </p>
        <p
          class="typo-normal"
        >
          Utflyttingsdato: 01.01.2025
        </p>
        <div
          class="c4"
        >
          <p
            class="typo-undertekst"
          >
            Endret 20.02.2010 av folkeregisteret (kilde: bruker)
          </p>
        </div>
        <div
          class="c4"
        >
          <p
            class="typo-undertekst"
          >
            Gyldig: 01.01.2020 – 01.01.2025
          </p>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
